src/app/admin/admin.module.ts

```diff
@@ -7,6 +7,14 @@ import { LearningObjectsComponent } from './pages/learning-objects/learning-obje
 import { AnalyticsComponent } from './pages/analytics/analytics.component';
 import { UsersComponent } from './pages/users/users.component';
 import { ContentWrapperComponent } from './components/content-wrapper/content-wrapper.component';
+import { FilterSearchComponent } from 'app/admin/components/filter-search/filter-search.component';
+import { FormsModule } from '@angular/forms';
```

Comment:
Are all of these imports necessary?

Reply:
@nvisal1 can you elaborate? They are all being used within the module, do you think they should be located at a different point in the module hierarchy?